tests/fixtures/s3.py

```diff
@@ -36,17 +42,17 @@ def __init__(self, image: str, settings: Settings) -> None:
         self.ports = {'9000/tcp': 9100, '9101/tcp': 9101}
         # It's necessary to have 4 volumes to enable versioning (run MinIO in Erasure Code)
         # https://docs.min.io/docs/minio-erasure-code-quickstart-guide.html
-        self.volumes = ['test-data-1:/data1', 'test-data-2:/data2', 'test-data-3:/data3', 'test-data-4:/data4']
+        volume_suffix = os.urandom(5).hex()
+        self.volumes = {f'test-data-{volume_suffix}-{i}': {'bind': f'/data{i}'} for i in range(1, 5)}
         self._command = 'minio server /data{1...4} --console-address ":9101"'
 
-    def start(self) -> None:
-        for i in range(1, 4):
-            self._docker.client.volumes.create('test-data-' + str(i))
-        super().start()
-
-    def stop(self, **kwargs) -> None:
+    def stop(self, **kwargs: any) -> None:
```

Comment:
Use `Any` type from `typing` package here.
Build-in `any` is not a type. It's a [function](https://docs.python.org/3/library/functions.html#any).
Please be more attentive to the code in my suggestions.

```suggestion
    def stop(self, **kwargs: Any) -> None:
```